src/Service/Sqs/SqsConsumer.php

```diff
@@ -92,7 +92,7 @@ public function handleSqs(SqsEvent $event, Context $context): void
                 $this->busDriver->putEnvelopeOnBus($this->bus, $envelope->with(...$stamps), $this->transportName);
             } catch (UnrecoverableExceptionInterface $exception) {
                 $this->logger->error(sprintf('SQS record with id "%s" failed to be processed. But failure was marked as unrecoverable. Message will be acknowledged.', $record->getMessageId()));
-                $this->logger->error($exception->getMessage());
+                $this->logger->error($exception);
```

Comment:
Does that work? (logging a whole object)

I was under the impression the log message must be a string. Have you tested it in Lambda?

Reply:
Yes, you can log the whole Exception. I wrote a composer script to replace the implementation with the same code as in this PR and got the expected result. These are screenshots from our Sentry log:

<img width="864" alt="Screenshot 2023-10-04 at 15 28 57" src="https://github.com/brefphp/symfony-messenger/assets/1649279/e5a0b427-c7c1-4732-9510-e45062e98251">

<img width="963" alt="Screenshot 2023-10-04 at 15 29 19" src="https://github.com/brefphp/symfony-messenger/assets/1649279/4c601bc7-7a32-47cc-9685-da5f821432c4">

Reply:
Hello, is it possible to review this PR again?